src/services/user/ThunderService.ts

```diff
@@ -0,0 +1,169 @@
+import {PostBaseResponseDto} from '../../interfaces/common/PostBaseResponseDto';
+import {ThunderCreateDto} from '../../interfaces/thunder/ThunderCreateDto';
+import {ThunderResponseDto} from '../../interfaces/thunder/ThunderResponseDto';
+import Thunder from '../../models/Thunder';
+import UserServiceUtils from '../user/UserServiceUtils';
+import ThunderServiceUtils from './ThunderServiceUtils';
+
+const createThunder = async (
+  thunderCreateDto: ThunderCreateDto,
+  userId: string,
+): Promise<PostBaseResponseDto> => {
+  await UserServiceUtils.findUserById(userId);
+
+  try {
+    const thunder = new Thunder({
+      title: thunderCreateDto.title,
+      deadline: new Date(thunderCreateDto.deadline),
+      hashtags: thunderCreateDto.hashtags,
+      content: thunderCreateDto.content,
+      limitMembersCnt: thunderCreateDto.limitMembersCnt,
+      members: [userId],
+      createdAt: Date.now(),
+      updatedAt: Date.now(),
+    });
+
+    await thunder.save();
+
+    const data = {
+      _id: thunder._id,
+    };
+
+    return data;
+  } catch (error) {
+    console.log(error);
+    throw error;
+  }
+};
+
+const findThunderAll = async (
+  userId: string,
+): Promise<ThunderResponseDto[] | []> => {
+  try {
+    await UserServiceUtils.findUserById(userId);
+
+    const thunderlist = await Thunder.find().sort({createdAt: 'asc'});
+
+    if (!thunderlist) {
+      return [];
+    }
+
+    const allThunder: ThunderResponseDto[] = [];
+
+    await Promise.all(
+      thunderlist.map(async (all: any) => {
```

Comment:
```suggestion
      thunderlist.map(async (thunder: any) => {
```
thunderList에서 thunder들을 하나하나 보는 것이므로, 
네이밍이 `all` 보다 `thunder`로 하는 것이 가독성이 더 좋을 것 같습니다..!!

Reply:
네! 변수명 고려하겠습니다